learn/getting-started/writing-a-grpc-service-with-ballerina.md

```diff
@@ -144,52 +176,68 @@ Running executable
+
+> **Note:** For more information on Ballerina projects, see [Organizing Ballerina code](/learn/organizing-ballerina-code/).
 
 ```bash
 $ bal new greeter_client
 ```
 
-> **Note:** For more information on creating Ballerina packages, see [Getting Started with Ballerina](/learn/getting-started-with-ballerina).
+You view the output below.
+
+```bash
+Created new package 'greeter_client' at greeter_service.
+```
+
+This creates a directory named `greeter_client` with the files below.
 
-Remove the automatically-created `main.bal` file as you are not going to use it in this guide.
+```bash
+.
+├── Ballerina.toml
+└── main.bal
+```
 
-#### Generating the source code
+>**Tip:** Remove the automatically-created `main.bal` file as you are not going to use it in this guide.
 
-Execute the following command inside the same directory in which you created the Ballerina project to generate the source code related to the service definition.
+### Generating the source code of the client
+
+In the terminal, from inside the same `grpc` directory, execute the command below to generate the source code related to the client definition.
 
 ```bash
 $ bal grpc --mode client --input helloworld.proto --output greeter_client/
 ```
 
-Once successfully executed, you could be able to view the below output.
-```
+Once successfully executed, you will see the output below.
+
+```bash
+Continuing with the existing protoc executor file at /var/folders/6p/8h4k83hj0r98n1s6mc3jy0480000gn/T/protoc-3.9.1-osx-x86_64.exe
 Successfully extracted the library files.
 Successfully generated the Ballerina file.
 ```
 
-Once you move into the directory, you will see the following two new files.
-```
-greeter_client
-├── - - -
+This creates two files below inside the `greeter_service` directory.
+
+```bash
+.
+├── Ballerina.toml
 ├── greeter_client.bal
 └── helloworld_pb.bal
 ```
 
-Here,
-- The `helloworld_pb.bal` file is the stub file that contains the classes that the client/service uses to talk to each
+- The `helloworld_pb.bal` file is the stub file, which contains the classes that the client/service uses to talk to each
 other and the Ballerina types corresponding to request and response messages.
-- The `greeter_client.bal` file is the client template file that contains the `main` function with the client declaration.
+- The `greeter_client.bal` file is the client template file, which contains the `main` function with the client declaration.
 
-#### Update and run the application
+### Updating the client template file
 
-Let’s update the client template file (i.e., `greeter_client.bal`) and write code to call the remote function in the server and get the server response back. Sample code is as follows,
+Replace the service template file (i.e., `greeter_client.bal`) with the code below to add the business logic to the remote method. In this case, you only need to update the `sayHello` method as shown below.
```

Comment:
Should be client template file

Reply:
Also this text is incorrect. It  is incorrect  to say `you only need to update the `sayHello` method as shown below.`

Reply:
Updated.